cmd/main.go

```diff
@@ -59,6 +59,8 @@ func setupFlags(name string) (*pflag.FlagSet, *options) {
 		"print format of test input")
 	flags.BoolVar(&opts.rawCommand, "raw-command", false,
 		"don't prepend 'go test -json' to the 'go test' command")
+	flags.BoolVar(&opts.ignoreNonJSONOutputLines, "ignore-non-json-output-lines", false,
+		"write non-JSON 'go test' output lines to stderr instead of failing")
```

Comment:
Hid (and updated the test baseline).

When I tried to put this in the readme, it seemed like that paragraph was getting long and the difference between the script's stderr vs. `gotestsum`'s stderr seemed muddy. So, I split the existing bit into two bullet points to try to make it a bit clearer what the new sentence is talking about. Otherwise kept it mostly the same:

> Note: when using `--raw-command`, the script must follow a few rules about stdout and stderr output:
> 
> * The stdout produced by the script must only contain the `test2json` output, or  `gotestsum` will fail. If it isn't possible to change the script to avoid  non-JSON output, you can use `--ignore-non-json-output-lines` to ignore
>   non-JSON lines and write them to `gotestsum`'s stderr instead.
> * Any stderr produced by the script will be considered an error (this behaviour  is necessary because package build errors are only reported by writting to  stderr, not the `test2json` stdout). Any stderr produced by tests is not  considered an error (it will be in the `test2json` stdout).